tests/Fixtures/e2e/Test_Frameworks/phpunit.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         bootstrap="./vendor/autoload.php"
+         colors="true"
+         convertErrorsToExceptions="true"
```

Comment:
I think a few of those options are either unnecessary or the default. If you open this in phpStorm and add the line `xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.4/phpunit.xsd"` (match the right version) and fetch the resource, you should be able to see which options are unnecessary